spec/features/state_file/complete_intake_spec.rb

```diff
@@ -47,6 +47,8 @@
       expect(page).to have_link "Download PDF"
       click_on "Show XML"
       expect(page.body).to include('efile:ReturnState')
+      expect(page.body).to include('<ABA_NMBR claimed="013456789"/>')
+      expect(page.body).to include('<BANK_ACCT_NMBR claimed="456789008765"/>')
```

Comment:
not sure this is really necessary since rails encryption is presumably tested at the framework level, and even the xml snippet "efile:ReturnState" we're already checking for is being encrypted now. but routing and account number are encrypted too outside of the data blob so i threw a check in there.